### ISSUE TITLE
View favorite tweets

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#58]
```
I would like to be able to see my favorite tweets in Pino. I guess this would 
fit in nicely in the “View” menu, next to “Timeline” and “Mentions”.

For me, viewing favorite tweets is more important than marking tweets as 
favorite (http://code.google.com/p/pino-twitter/issues/detail?id=57). This is 
because, on the go, using a smartphone, I mark those tweets as favorites on 
which I want to follow up later on.
```

Original issue reported on code.google.com by `alex.bepple` on 31 Jan 2010 at 9:40

**Comment by GoogleCodeExporter** [comment]
Original comment by `tro...@gmail.com` on 31 Jan 2010 at 10:37
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Closed by 740d551b56 revision.
```

Original comment by `tro...@gmail.com` on 30 Mar 2010 at 3:20
- Changed state: **Fixed**
